src/layman/map/micka/csw.py

```diff
@@ -141,7 +139,7 @@ def csw_insert(username, mapname):
     return muuid
 
 
-def map_json_to_operates_on(map_json):
+def map_json_to_operates_on(map_json, operates_on_muuids_filter=None, editor=None):
```

Comment:
Tato metoda mi přijde dlouhá a složitá. Pobavmež se, jeslti nejde roztrhnout na víc.